client/src/browser-table.html

```diff
@@ -279,7 +279,7 @@ <h1 label>Browser support</h1>
       </div>
       <div class="row">
         <a spec href="https://html.spec.whatwg.org/multipage/webappapis.html#integration-with-the-javascript-module-system" target="_blank" rel="noopener">
-          <div label='<script type="module">'><svg viewBox="0 0 26 22"><use xlink:href="/sprite.octicons.svg#import"></use></svg></div>
+          <div label='ES Modules'><svg viewBox="0 0 26 22"><use xlink:href="/sprite.octicons.svg#import"></use></svg></div>
```

Comment:
I’ve never actually seen these be called <q>ES Modules</q>, only <q>ES6 Modules</q>, also the label attribute should use `"` instead of `'`.

Reply:
@ExE-Boss See e.g. https://hacks.mozilla.org/2018/03/es-modules-a-cartoon-deep-dive/ and https://medium.com/web-on-the-edge/es-modules-in-node-today-32cff914e4b. See also https://bytearcher.com/articles/es6-vs-es2015-name/, https://benmccormick.org/2015/09/14/es5-es6-es2016-es-next-whats-going-on-with-javascript-versioning/ and https://codeburst.io/javascript-wtf-is-es6-es8-es-2017-ecmascript-dca859e4821c for more info. :wink: